app/src/main/java/com/kickstarter/viewmodels/EnvironmentalCommitmentsViewHolderViewModel.kt

```diff
@@ -55,7 +56,6 @@ interface EnvironmentalCommitmentsViewHolderViewModel {
 
         override fun onCleared() {
             disposables.clear()
-            super.onCleared()
         }
```

Comment:
onCleared came from extending viemodel, removing that extension also means you needed to add this to the inputs

Reply:
This is the input function that is being called by the viewholder. Take a look at line 34 in the EnvironmentalCommitmentsViewHolder.kt in this pull request